examples/post_training_quantization/onnx/mobilenet_v2/main.py

```diff
@@ -61,13 +62,17 @@ def validate(path_to_model: Path, validation_loader: torch.utils.data.DataLoader
     return accuracy_score(predictions, references)
 
 
-def run_benchmark(path_to_model: Path, shape: Optional[List[int]] = None, verbose: bool = True) -> float:
-    command = f"benchmark_app -m {path_to_model} -d CPU -api async -t 15"
-    if shape is not None:
-        command += f' -shape [{",".join(str(x) for x in shape)}]'
-    cmd_output = subprocess.check_output(command, shell=True)  # nosec
-    if verbose:
-        print(*str(cmd_output).split("\\n")[-9:-1], sep="\n")
+def run_benchmark(path_to_model: Path, shape: List[int]) -> float:
+    command = [
+        "benchmark_app",
+        "-m", path_to_model.as_posix(),
+        "-d", "CPU",
+        "-api", "async",
+        "-t", "15",
+        "-shape", str(shape),
+    ]  # fmt: skip
+    cmd_output = subprocess.check_output(command, text=True)
+    print(*cmd_output.splitlines()[-8:], sep="\n")
```

Comment:
Dont see any depends print function from type of used progress bar. 
If you think that this changes is required, you are free to create PR.

Reply:
I do not agree. Let's align it and use here and below `nncf_logger` instead of `print`s as you did for `track` instead of `tqdm`.

Reply:
What do you disagree with?

Reply:
I disagree with this:

> Dont see any depends print function from type of used progress bar.

I see that we should be consistent. If you're adding a progress bar from the NNCF logging part, thus add please a logging from NNCF instead of `print`s.

Reply:
As i said before, if you are sure that using nncf_logger can improve examples for any way, you are free to create your own PR.

But i sure that nncf_logger is not should be used in examples, it's only increase output without adding any useful information for user.

@alexsu52 what do you think about it?

Reply:
@AlexanderDokuchaev, sorry, but your PR was stated as `Improve examples`. But proposed changes introduce more mess than an improvement in terms of the NNCF API usage. I don't see any reason to replace `tqdm` with the `track` then.
Please, revert the changes or align API usage everywhere.

Reply:
> Please, revert the changes or align API usage everywhere.

No, I do not agree. 

Please be consistent in your discussion, your are start this thread about replacing `print` to `nncf_logger`, Now you ask to revert change by reason that you dont see any reasons in replacing tqdm to track.

If you ask me about why `track` is more preferable then `tqdm`, i can answer:
1. Output for `tqdm` in CI or if save to file, looks too massive because each update of progress bar generate new line.
2. For tqdm can be disable but in this case output does not contain progress bar, that contain number of iterations and time, but it can be used to debug and detect issues.
3. Track based on rich, that resolve this issue and saves progress bar in one line.

To see it, you can reproduce output localy, 

```python
# script.py
from tqdm import tqdm
from time import sleep
from nncf.common.logging.track_progress import track

l = [1,2,3,4,5]
print("TQDM:")
for _ in tqdm(l):
    sleep(.5)
print("TRACK:")
for _ in track(l):
    sleep(.5)
```

run command:
```bash
python script.py  > log 2>&1
```
Output: 
```
INFO:nncf:NNCF initialized successfully. Supported frameworks detected: torch, onnx, openvino
TQDM:

  0%|          | 0/5 [00:00<?, ?it/s]
 20%|██        | 1/5 [00:00<00:02,  2.00it/s]
 40%|████      | 2/5 [00:01<00:01,  2.00it/s]
 60%|██████    | 3/5 [00:01<00:01,  2.00it/s]
 80%|████████  | 4/5 [00:02<00:00,  2.00it/s]
100%|██████████| 5/5 [00:02<00:00,  2.00it/s]
100%|██████████| 5/5 [00:02<00:00,  2.00it/s]
TRACK:
Working... ━━━━━━━━━━━━━━━━━━ 100% 5/5 • 0:00:02 • 0:00:00
```

Reply:
In general, we should not overload NNCF examples with additional NNCF functionality like logging as we want users to focus on concrete API which is demonstrated in the example. So I don't think we should replace 'print's by NNCF logging for "the consistency". Regarding 'track', I agree it reduces the output but this is again NNCF functionality which doesn't belong to API that we want to demonstrate here and I would try to avoid it. @AlexanderDokuchaev , is there a way to use 'rich' directly here?
cc @alexsu52

Reply:
It is possible, but default rich progress bar does not contain number of iteration and show only left rime,
and combination of two style of progress bar in one script looks weird 

default rich:
<img width="1068" alt="image" src="https://github.com/user-attachments/assets/e2398038-ff1f-41b5-a69f-dcfbc18a4fb5">
nncf track:
<img width="1096" alt="image" src="https://github.com/user-attachments/assets/47ae0b94-e8ca-4439-87e2-07c034bf4130">

Reply:
I would suggest to use the default rich.

Reply:
https://github.com/openvinotoolkit/nncf/pull/3091/commits/23d8ba342fa9ade9bfedb6918f9678a7bdccb6f5
https://github.com/openvinotoolkit/nncf/actions/runs/11975030346